drodrpg/DRODLib/Character.cpp

```diff
@@ -2443,7 +2443,10 @@ void CCharacter::Process(
 					if (roomID)
 					{
 						const bool bMarkExplored = command.w != 0;
-						pGame->AddRoomToMap(roomID, bMarkExplored ? MapState::Explored : MapState::NoDetail);
+						MapState mapState = bMarkExplored ? MapState::Explored : MapState::NoDetail;
```

Comment:
I like how you're clarifying the code with these simple refactorings.

Make mapState const for these kinds of var declarations. The logic is easier to reason about when it's explicit that vars are immutable.